lib/trln_argon/bookmarks_controller_behavior.rb

```diff
@@ -49,6 +53,15 @@ def sms_path(options = {})
       def sms_url(options = {})
         sms_bookmarks_url(options)
       end
+
+      def bookmarks_share_url
+        bookmarked_ids = ''
+        current_bookmarks.each do |bookmark|
+          bookmarked_ids = bookmarked_ids + bookmark['document_id'].to_s + '|'
+        end
+        bookmarked_ids = bookmarked_ids.gsub(/\|+$/, '') # remove trailing |
+        '/?doc_ids=' + bookmarked_ids
```

Comment:
There is a bug here. If you bookmark a record from the TRLN view that is not in your location collection the link generated does not display the non-local record in the results. Let's talk about how to handle this. It's also preferable to use Rails URL helpers for generating links in the application instead of hard-coding them as strings.